modules/currency-l0/src/main/scala/org/tessellation/currency/l0/modules/Programs.scala

```diff
@@ -22,7 +23,7 @@ import org.tessellation.security.{Hasher, SecurityProvider}
 
 object Programs {
 
-  def make[F[_]: Async: Random: KryoSerializer: Hasher: SecurityProvider](
+  def make[F[_]: Async: Random: KryoSerializer: JsonHashSerializer: Hasher: SecurityProvider](
```

Comment:
I think a better name is needed, as it's not only for Hash now

Reply:
In my mind, "serialization" implies binary as one side of the Encode/Decode functionality. Perhaps `JsonSerializer` is enough to imply the intended behavior.